### PR TITLE
Reformat python files

### PR DESCRIPTION
**Pull request by banasraf**
Signed-off-by: Rafal <Banas.Rafal97@gmail.com>

## Category:

Refactoring

## Description:

Fixing python linter errors on some of the Python tests.

### Affected modules and functionalities:

Python tests

### Key points relevant for the review:

Formatting and correctness

### Tests:

- [ ] Existing tests apply
- [x] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [x] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: DALI-2825

### CODE REVIEW

**Comment by banasraf** [comment]
!build

File: dali/test/python/test_external_source_pytorch_cpu.py [hunk 1]
```diff
@@ -15,7 +15,8 @@
...
 # nose will query for the methods available and will run them
 # the test_internals_operator_external_source is 99% the same for cupy and numpy tests
-# so it is better to store everything in one file and just call `use_cupy` to switch between the default numpy and cupy
+# so it is better to store everything in one file and just call `use_cupy`
+# to switch between the default numpy and cupy
 
-from test_external_source_impl import *
-use_torch(False)
+from test_external_source_impl import *  # noqa: F403
+use_torch(False)  # noqa: F405
```
Comment (szalpal) [inline dali/test/python/test_external_source_pytorch_cpu.py:22]: Could it be like this?
```suggestion
from test_external_source_impl import use_torch
use_torch(False)  # noqa: F405
```

File: dali/test/python/test_operator_transpose.py [hunk 2]
```diff
@@ -91,8 +84,10 @@ def check_transpose_vs_numpy(device, batch_size, dim, total_volume, permutation)
     print("permutation ", permutation)
     eii1 = RandomlyShapedDataIterator(batch_size, max_shape=max_shape)
     eii2 = RandomlyShapedDataIterator(batch_size, max_shape=max_shape)
-    compare_pipelines(TransposePipeline(device, batch_size, "", iter(eii1), permutation=permutation),
-                      PythonOpPipeline(lambda x: transpose_func(x, permutation), batch_size, "", iter(eii2)),
+    compare_pipelines(TransposePipeline(device, batch_size, "", iter(eii1),
+                                        permutation=permutation),
+                      PythonOpPipeline(lambda x: transpose_func(x, permutation),
+                                       batch_size, "", iter(eii2)),
```
Comment (szalpal) [inline dali/test/python/test_operator_transpose.py:90]:
```suggestion
    compare_pipelines(
        TransposePipeline(device, batch_size, "", iter(eii1), permutation=permutation),
        PythonOpPipeline(lambda x: transpose_func(x, permutation), batch_size, "", iter(eii2)),
```

File: dali/test/python/test_fw_iterators.py [hunk 3]
```diff
@@ -125,12 +130,17 @@ def test_mxnet_iterator_last_batch_no_pad_last_batch():
...
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
+                             num_gpus=num_gpus, data_paths=data_sets[0],
+                             random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:139]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=False),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 4]
```diff
@@ -94,8 +96,11 @@ def create_test_pipeline(batch_size, num_threads, device_id, num_gpus, data_path
         pipe.set_outputs(labels)
         return pipe
 
-    pipes, _ = create_pipeline(lambda gpu: create_test_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu, num_gpus=num_gpus,
-                                                                data_paths=image_data_set), batch_size, num_gpus)
+    pipes, _ = create_pipeline(
+        lambda gpu:
+        create_test_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu,
+                             num_gpus=num_gpus, data_paths=image_data_set), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:103]:
```suggestion
    pipes, _ = create_pipeline(
        lambda gpu: create_test_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu,
                                         num_gpus=num_gpus, data_paths=image_data_set),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py (hunk 3 above)
Comment (szalpal) [reply dali/test/python/test_fw_iterators.py:139]: Github UI makes this one (and the next ones) really bad, but I feel this way is better. Please compare in your IDE. I'm referring to putting `batch_size` and `num_gpus` at the indentation point of the lambda. This way it's clearer, these are args to `create_pipeline`.
image

File: dali/test/python/test_fw_iterators.py [hunk 5]
```diff
@@ -200,12 +210,16 @@ def test_mxnet_iterator_last_batch_pad_last_batch():
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:218]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=True),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 6]
```diff
@@ -230,11 +244,15 @@ def test_mxnet_iterator_not_fill_last_batch_pad_last_batch():
...
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:252]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=True),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 7]
```diff
@@ -365,43 +388,54 @@ def check_mxnet_iterator_pass_reader_name(shards_num, pipes_number, batch_size,
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:438]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=False),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 8]
```diff
@@ -420,12 +454,16 @@ def test_gluon_iterator_last_batch_pad_last_batch():
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:462]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=True),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 9]
```diff
@@ -450,9 +488,12 @@ def test_gluon_iterator_not_fill_last_batch_pad_last_batch():
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=False, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=False, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:496]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=False, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=True),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 10]
```diff
@@ -553,37 +605,45 @@ def check_gluon_iterator_pass_reader_name(shards_num, pipes_number, batch_size,
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:646]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=False),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 11]
```diff
@@ -602,9 +662,12 @@ def test_pytorch_iterator_last_batch_pad_last_batch():
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:670]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=True),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 12]
```diff
@@ -632,9 +695,12 @@ def test_pytorch_iterator_not_fill_last_batch_pad_last_batch():
...
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=False, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                             data_paths=data_sets[0], random_shuffle=False, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=True), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:703]:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=False, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=True),
        batch_size, num_gpus)
```

File: dali/test/python/test_fw_iterators.py [hunk 13]
```diff
@@ -679,13 +745,15 @@ def create_custom_pipeline(batch_size, num_threads, device_id, num_gpus, data_pa
...
     import torch
     num_gpus = 1
     batch_size = 100
-    pipes, _ = create_pipeline(lambda gpu: create_custom_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu, num_gpus=num_gpus,
-                                                                  data_paths=image_data_set), batch_size, num_gpus)
+    pipes, _ = create_pipeline(
+        lambda gpu:
+        create_custom_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu,
+                               num_gpus=num_gpus, data_paths=image_data_set), batch_size, num_gpus
+    )
```
Comment (szalpal) [inline dali/test/python/test_fw_iterators.py:756]:
```suggestion
    pipes, _ = create_pipeline(
        lambda gpu: create_custom_pipeline(batch_size=batch_size, num_threads=4, device_id=gpu,
                                           num_gpus=num_gpus, data_paths=image_data_set),
        batch_size, num_gpus)
```

Comment (szalpal) [reply dali/test/python/test_fw_iterators.py:756]: ... and the remaining ones in this file. Would you mind applying my suggestion to the rest of those?

File: dali/test/python/test_external_source_pytorch_cpu.py (hunk 1 above)
Comment (banasraf) [reply dali/test/python/test_external_source_pytorch_cpu.py:22]: No. This file needs to import all test functions from the test_external_source_impl module. Above those lines there's comment that explains this

**Review by szalpal — APPROVED** [review]

**Review by prak-nv — APPROVED** [review]

**Comment by banasraf** [comment]
!build

**Pull request merged**